dhctl/cmd/dhctl/main.go

```diff
@@ -40,6 +41,55 @@ import (
 	"github.com/deckhouse/deckhouse/dhctl/pkg/util/tomb"
 )
 
+var (
+	allowedCommands                         []string
```

Comment:
i think we may make register commands more clearly,  something like:
```
type Command struct {
    Name  string
    Help  string
    DefineFunc func(*kingpin.CmdClause)
}

var commandsList = []Command{
    {
        Name: "bootstrap",
        Help: "Bootstrap cluster.",
        DefineFunc: func(cmd *kingpin.CmdClause) {
            bootstrap.DefineBootstrapCommand(kpApp, cmd)
        },
    },
    ...
}

```
and then we may check and register each command in one range loop:

```
func registerCommands(kpApp *kingpin.Application, allowedCommands []string, commandsList []Commandd) {
  for _, cmdInfo := range commandsList {
      if allowed, subcommands := checkCommand(cmdInfo.Name, allowedCommands); allowed {
          cmd := kpApp.Command(cmdInfo.Name, cmdInfo.Help)
          if cmdInfo.DefineFunc != nil {
              cmdInfo.DefineFunc(cmd)
          }
          // registerSubcommands(cmd, subcommands) 
      }
  }
}
```
 could you try do it?

Reply:
Done